shared/reports/enums.py

```diff
@@ -7,6 +7,9 @@ class UploadState(CodecovDatabaseEnum):
     ERROR = (3,)
     FULLY_OVERWRITTEN = (4,)
     PARTIALLY_OVERWRITTEN = (5,)
+    PARALLEL_PROCESSED = (
+        6,
+    )  # not used right now, but will when parallel upload procesing is rolled out
```

Comment:
what is the difference between this and `PROCESSED`? could you explain when this status will be used in a comment?